build/dependencies.props

```diff
@@ -8,6 +8,7 @@
     <MicrosoftSourceLinkGitHubVersion>1.0.0</MicrosoftSourceLinkGitHubVersion>
     <NetStandardVersion>2.0.3</NetStandardVersion>
     <NewtonsoftVersion>13.0.3</NewtonsoftVersion>
+    <SystemDiagnosticSourceVersion>8.0.1</SystemDiagnosticSourceVersion>
```

Comment:
@GeoK Is this the minimum version we can use? I remember we had issues with this when we added it to MSAL because Azure Functions didn't have this dependency. (I think it was in .NET 6 Azure Functions, so we downgraded to 6.0.1.)

Reply:
Yep, we have to lower this.